packages/notification/src/server.ts

```diff
@@ -70,7 +70,7 @@ export async function createServer() {
   async function start() {
     await server.start()
     await database.start()
-    loopNotificationQueue(server)
+    loopNotificationQueue('user@gmail.com', server)
```

Comment:
Could we instead of passing an example email address here, store the primary recipient email in the database at the time of writing the `NotificationQueue`?